src/include/sof/trace/trace.h

```diff
@@ -435,42 +439,59 @@ struct tr_ctx {
 
 /* tracing from infrastructure part */
 
-#define tr_err(ctx, fmt, ...) \
-	trace_error_with_ids(_TRACE_INV_CLASS, ctx, \
-			     _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
 #define tr_err_atomic(ctx, fmt, ...) \
 	trace_error_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
 				    _TRACE_INV_ID, _TRACE_INV_ID, \
 				    fmt, ##__VA_ARGS__)
 
-#define tr_warn(ctx, fmt, ...) \
-	trace_warn_with_ids(_TRACE_INV_CLASS, ctx, \
-			    _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
 #define tr_warn_atomic(ctx, fmt, ...) \
 	trace_warn_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
 				   _TRACE_INV_ID, _TRACE_INV_ID, \
 				   fmt, ##__VA_ARGS__)
 
-#define tr_info(ctx, fmt, ...) \
-	trace_event_with_ids(_TRACE_INV_CLASS, ctx, \
-			     _TRACE_INV_ID, _TRACE_INV_ID, fmt, ##__VA_ARGS__)
-
 #define tr_info_atomic(ctx, fmt, ...) \
 	trace_event_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
 				    _TRACE_INV_ID, _TRACE_INV_ID, \
 				    fmt, ##__VA_ARGS__)
 
+#define tr_dbg_atomic(ctx, fmt, ...) \
+	tracev_event_atomic_with_ids(_TRACE_INV_CLASS, ctx, \
+				     _TRACE_INV_ID, _TRACE_INV_ID, \
+				     fmt, ##__VA_ARGS__)
+
+#ifdef __ZEPHYR__
```

Comment:
ok, best to have a comment here say this is tmp until all users have been updated.